### PR TITLE
Create XanderRubio.json

### PR DESCRIPTION
**Pull request by XanderRubio**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation as needed.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

I've added the JSON file to add my links to the project. Thank you for this great project. Let me know if I need to make adjustments, and have a great day!

### CODE REVIEW

File: BioDrop/data/XanderRubio.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Alexander Clemens",
+    "bio": "Software Engineer | Open Source Maintainer | Colombia 🇨🇴",
+    "links": [
+        {
+            "name": "LinkedIn",
+            "url": "https://www.linkedin.com/in/alexanderclemens/",
+            "icon": "FaLinkedIn"
```
Comment (sital002) [inline BioDrop/data/XanderRubio.json:8]:
```suggestion
            "icon": "FaLinkedin"
```
FaLinkedIn icon isnot correct it must be either FaLinkedin or FaLinkedInin You can search for icons here https://www.biodrop.io/icons

**Review by sital002 — CHANGES_REQUESTED** [review]
Your json file must be in data folder instead of ```BioDrop/data/XanderRubio.json``` It must be ```data/XanderRubio.json```.

File: BioDrop/data/XanderRubio.json (hunk 1 above)
Comment (XanderRubio) [reply BioDrop/data/XanderRubio.json:8]: Understood making the fix now.

**Review by sital002 — APPROVED** [review]
Looks good 👍
Welcome to Eddiehub community 🎉🎉

**Comment by XanderRubio** [comment]
Updates have been made from comments on fixing the FaLinkedin name and renaming the file structure to `data/XanderRubio.json`. Thank you for providing your feedback!

**Comment by XanderRubio** [comment]
Thank you @sital002! Have a nice day!

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for Contributing! 🎉

**Comment by XanderRubio** [comment]
Thank you @kumarsonsoff3. Have a great day!

**Pull request merged**